lib/elixir/test/elixir/code_test.exs

```diff
@@ -153,6 +153,36 @@ defmodule CodeTest do
                {:error, {1, "unsafe atom does not exist: ", "there_is_no_such_atom"}}
     end
 
+    test "supports static_atoms_encoder" do
+      ref = make_ref()
+
+      encoder = fn "there_is_no_such_atom", metadata ->
```

Comment:
Nitpick, but maybe for better assertion errors we can have:

```elixir
encoder = fn atom, meta ->
  assert atom == "ther_is_no_such_atom"
  assert meta[:line] == =1
  # ...same
end
```

Reply:
Addressed this in 23e3b95